typings/index.flow.js

```diff
@@ -202,6 +202,11 @@ interface UnsafeByPropsQueries {
     | Array<ReactTestInstance>
     | [];
 }
+
+interface ByRoleOption {
```

Comment:
Plural `options` in order to match type parameter naming

```suggestion
interface ByRoleOptions {
```